tests/conftest.py

```diff
@@ -211,3 +224,150 @@ def block_meteorite_emission():
 @pytest.fixture
 def fn():
     return PiecewiseConstantFunction(1)
+
+
+@pytest.fixture
+def pod1():
```

Comment:
+1, though I recognize that these are just being moved from one file to another.

Honestly, conftest.py bothers me a bit -- generally speaking in python you can always tell where a particular variable comes from by searching for it within the same file (it's either defined or imported somewhere in the file). Pytest fixtures usually follow that pattern too, unless they're defined in conftest.py. Then you just have to know enough about pytest to remember to look there. (Or `git grep` the codebase for the name, in which case descriptive names are helpful.)

The conftest ship has sailed in this codebase, though, and I don't see a great option for avoiding it here while still being able to share the fixture among multiple test files.

Reply:
I cheated on this a little bit.  Instead of renaming everything I just moved the `pod1` definition to conftest and renamed it `simple_pod`, and left everything else alone.  I agree that these should maybe have better names but I think it's actually easier to just leave them alone for now, and handle them on a case-by-case basis in the future.